README.rst

```diff
@@ -127,6 +127,13 @@ As with normal page serving, the serving of shared pages continues to respect Wa
 
 This project adds these additional hooks:
 
+``before_route_page``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Called when routing, before a page's ``route()`` method is called. This hook is passed the ``ServeView``, the ``request``, and the ``page`` that will have ``route()`` called on it. If the callable returns an ``HttpResponse``, that response will be returned immediately to the user.
+
+This hook allows for any necessary customization of Wagtail's built-in routing behavior, for example to support ShareableRoutablePageMixin (with link to below section of documentation).
```

Comment:
> (with link to below section of documentation)

Is this a note to self?

Reply:
Yup, I think it was. That's been changed to a link in b24956c.